cli/tests/unit/request_test.ts

```diff
@@ -62,3 +62,48 @@ unitTest(async function cloneRequestBodyStream(): Promise<void> {
 
   assertEquals(b1, b2);
 });
+
+unitTest(function requiresOneArgument() {
```

Comment:
I think we're missing a test case for when a Request has been passed to the Request constructor, is there anything else missing?